src/app/components/loading/loading.component.scss

```diff
@@ -0,0 +1,5 @@
+.loading {
+  margin: 0 auto;
+  opacity: .7;
```

Comment:
Properties should be ordered margin, text-align, opacity

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)